Source/Renderer/FramebufferManager.js

```diff
@@ -272,9 +295,6 @@ FramebufferManager.prototype.destroyResources = function () {
     this._depthTexture = undefined;
   }
 
-  if (defined(this._framebuffer)) {
-    this._framebuffer.destroy();
-    this._framebuffer = undefined;
-  }
+  this.destroyFramebuffer();
```

Comment:
I may revisit this - I think it's useful to separate the code to destroy the framebuffer and attachmenets into two functions, and it might be cleaner if `destroyResources` becomes `destroyAttachments` and doesn't always call `destroyFramebuffer` at the end.